### PR TITLE
EZP-29049: As a user, I want to delete a single item in the trash.

### PR DESCRIPTION
**Pull request by mikadamczyk**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-29049
| Bug fix?      |no
| New feature?  | yes
| BC breaks?    |no
| Tests pass?   | yes
| Doc needed?   |no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

There is no way to delete a single item (or multiple items) in the trash. There is only one button: Empty trash.

screen shot 2018-04-03 at 3 39 35 pm

screen shot 2018-04-03 at 3 36 36 pm

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/scripts/admin.trash.list.js [hunk 1]
```diff
@@ -31,15 +31,20 @@
...
     const buttonRestore = document.querySelector('#trash_item_restore_restore');
     const buttonRestoreUnderNewParent = document.querySelector('#trash_item_restore_location_select_content');
+    const buttonDelete = document.querySelector('#delete-trash-items');
 
     const enableButtons = (event) => {
+        const deleteCheckbox = document.querySelector('form[name="trash_item_delete"] input[type="checkbox"][value="' + event.target.value + '"]');
         const isNonEmptySelection = checkboxes.some(el => el.checked);
         const isMissingParent = checkboxes.some(el => el.checked && parseInt(el.dataset.isParentInTrash, 10) === 1);
 
+        if (deleteCheckbox !== null) {
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.trash.list.js]: `if (deleteCheckbox) {` is enough.

File: src/bundle/Resources/views/admin/trash/list.html.twig [hunk 2]
```diff
@@ -83,6 +97,18 @@
                     {{ form_widget(form_trash_item_restore._token) }}
                     {{ form_end(form_trash_item_restore, { 'render_rest': false }) }}
                 </div>
+
+                {{ form_start(form_trash_item_delete, {
+                    'action': path('ezplatform.trash.delete')
+                }) }}
+                {% for key, form in form_trash_item_delete.trash_items %}
+                {% set trash_item = trash_items[key] %}
+                        {{ form_widget(form) }}
```
Comment (dew326) [inline src/bundle/Resources/views/admin/trash/list.html.twig]: indentation

File: src/bundle/Controller/TrashController.php [hunk 3]
```diff
@@ -240,4 +266,48 @@ public function restoreAction(Request $request): Response
...
+     *
+     * @throws \Symfony\Component\OptionsResolver\Exception\InvalidOptionsException
+     * @throws \Symfony\Component\Translation\Exception\InvalidArgumentException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     * @throws \LogicException
+     * @throws \InvalidArgumentException
+     */
+    public function deleteAction(Request $request): Response
+    {
+        if ($this->isGranted(new Attribute('content', 'remove'))) {
```
Comment (webhdx) [inline src/bundle/Controller/TrashController.php]: Use early return/redirection in this case.

Comment (webhdx) [inline src/bundle/Controller/TrashController.php]: Use early return/redirection in this case.

**Pull request merged**